### ISSUE TITLE
Running .love files through loveftw

### ISSUE THREAD

**Issue by desk467** [ahrnbom/l-veftw#1]
It would be very nice if love-ftw could execute .love files from an installation in  w10m. Thus it would be possible that anyone could run .love games from your phone without having to create a compilation of this project, facilitating debugging.

**Comment by ahrnbom** [comment]
Yes, absolutely! My first priority will be to make it work as a "release" mode, which doesn't do that (you can always develop your game on your computer, and just test it a few times on your phone).

If that ends up working well, I'll look into making a more dev-friendly version (perhaps on a separate branch?). The way I think it could be implemented is that the app could be associated with .love files, so that tapping a .love file in any context on your phone (or PC or whatever) will open it in LöveFTW.

It's still quite early in development so we'll see how it goes. But I absolutely agree that this is a good idea!

Best regards, and thanks for following the (somewhat slow) development of LöveFTW :)

**Comment by desk467** [comment]
Awesome :)
I hope one day to be able to contribute with this..

Btw.. how can I create my .appx with this proj ?

**Comment by ahrnbom** [comment]
There will be instructions on the README once it's in a usable state (which should be quite soon, I think)

**Comment by ahrnbom** [comment]
A seemingly working implementation of running .love files with LöveFTW by just double-clicking on them is now in place. And it doesn't prevent you from pre-including any games, LöveFTW just launches the pre-included game if no .love file is the reason the app was started (if no pre-included game exists, it shows the "no game" screen like you'd expect).
